src/bin/cargo/commands/info.rs

```diff
@@ -0,0 +1,39 @@
+use cargo::ops::cargo_info::info;
+use cargo::util::command_prelude::*;
+use cargo_util_schemas::core::PackageIdSpec;
+
+pub fn cli() -> Command {
```

Comment:
Do we have a "new command proposal" somewhere for us to have an FCP on?  For cargo add, it was the PR description itself, see https://github.com/rust-lang/cargo/pull/10472

Looks like we had our insta-stable conversation at https://github.com/rust-lang/cargo/pull/10472#issuecomment-1076778255

This is a bit different of a case because there is a command unrelated to this effort currently that we'd be shadowing.

Reply:
> This is a bit different of a case because there is a command unrelated to this effort currently that we'd be shadowing.

Do you mean https://crates.io/crates/cargo-info?

Reply:
Correct.  It might not be a blocker but we need to make sure we call it out

Reply:
I have added it to the PR description.